src/media/mic.rs

```diff
@@ -16,10 +19,35 @@ use cpal::{traits::StreamTrait, Sample, Stream};
 
 use crossbeam_channel::{Receiver, TryRecvError};
 
+#[cfg(not(test))]
+mod private {
```

Comment:
I'm reasonably sure this setup is safe.
cpal tells us a `Stream` cannot be moved across threads (it is not `Send`) due to some platform constraints. However, I want to move an `Arc<....<Stream>>` into the render thread so the control thread Microphone object can go out of scope while the Stream should continue to run. This private mod prevents any calls (e.g. pause/resume) on the stream so this should make it safe to `Send`